src/data/balance/starters.ts

```diff
@@ -0,0 +1,1224 @@
+  [Species.MUNKIDORI]: Abilities.NEUROFORCE,
+  [Species.FEZANDIPITI]: Abilities.LEVITATE,
+  [Species.OGERPON]: Abilities.OPPORTUNIST,
+  [Species.GOUGING_FIRE]: Abilities.BEAST_BOOST,
+  [Species.RAGING_BOLT]: Abilities.BEAST_BOOST,
+  [Species.IRON_BOULDER]: Abilities.SHARPNESS,
+  [Species.IRON_CROWN]: Abilities.SHARPNESS,
+  [Species.TERAPAGOS]: Abilities.SOUL_HEART,
+  [Species.PECHARUNT]: Abilities.TOXIC_CHAIN,
+  [Species.ALOLA_RATTATA]: Abilities.ADAPTABILITY,
+  [Species.ALOLA_SANDSHREW]: Abilities.ICE_SCALES,
+  [Species.ALOLA_VULPIX]: Abilities.SHEER_FORCE,
+  [Species.ALOLA_DIGLETT]: Abilities.STURDY,
+  [Species.ALOLA_MEOWTH]: Abilities.DARK_AURA,
+  [Species.ALOLA_GEODUDE]: Abilities.DRY_SKIN,
+  [Species.ALOLA_GRIMER]: Abilities.TOXIC_DEBRIS,
+  [Species.ETERNAL_FLOETTE]: Abilities.MAGIC_GUARD,
+  [Species.GALAR_MEOWTH]: Abilities.STEELWORKER,
+  [Species.GALAR_PONYTA]: Abilities.MOXIE,
+  [Species.GALAR_SLOWPOKE]: Abilities.UNAWARE,
+  [Species.GALAR_FARFETCHD]: Abilities.INTREPID_SWORD,
+  [Species.GALAR_ARTICUNO]: Abilities.SERENE_GRACE,
+  [Species.GALAR_ZAPDOS]: Abilities.TOUGH_CLAWS,
+  [Species.GALAR_MOLTRES]: Abilities.DARK_AURA,
+  [Species.GALAR_CORSOLA]: Abilities.SHADOW_SHIELD,
+  [Species.GALAR_ZIGZAGOON]: Abilities.POISON_HEAL,
+  [Species.GALAR_DARUMAKA]: Abilities.FLASH_FIRE,
+  [Species.GALAR_YAMASK]: Abilities.TABLETS_OF_RUIN,
+  [Species.GALAR_STUNFISK]: Abilities.ARENA_TRAP,
+  [Species.HISUI_GROWLITHE]: Abilities.RECKLESS,
+  [Species.HISUI_VOLTORB]: Abilities.TRANSISTOR,
+  [Species.HISUI_QWILFISH]: Abilities.MERCILESS,
+  [Species.HISUI_SNEASEL]: Abilities.SCRAPPY,
+  [Species.HISUI_ZORUA]: Abilities.ADAPTABILITY,
+  [Species.PALDEA_TAUROS]: Abilities.ADAPTABILITY,
+  [Species.PALDEA_WOOPER]: Abilities.THICK_FAT,
+  [Species.BLOODMOON_URSALUNA]: Abilities.BERSERK
+};
+
+const starterCandyCosts: { passive: integer; costReduction: [integer, integer]; egg: integer; }[] = [
+  { passive: 40, costReduction: [25, 60], egg: 30 }, // 1 Cost
+  { passive: 40, costReduction: [25, 60], egg: 30 }, // 2 Cost
+  { passive: 35, costReduction: [20, 50], egg: 25 }, // 3 Cost
+  { passive: 30, costReduction: [15, 40], egg: 20 }, // 4 Cost
+  { passive: 25, costReduction: [12, 35], egg: 18 }, // 5 Cost
+  { passive: 20, costReduction: [10, 30], egg: 15 }, // 6 Cost
+  { passive: 15, costReduction: [8, 20], egg: 12 }, // 7 Cost
+  { passive: 10, costReduction: [5, 15], egg: 10 }, // 8 Cost
+  { passive: 10, costReduction: [5, 15], egg: 10 }, // 9 Cost
+  { passive: 10, costReduction: [5, 15], egg: 10 }, // 10 Cost
+];
+
+export function getPassiveCandyCount(baseValue: integer): integer {
+  return starterCandyCosts[baseValue - 1].passive;
+}
+
+export function getValueReductionCandyCounts(baseValue: integer): [integer, integer] {
+  return starterCandyCosts[baseValue - 1].costReduction;
+}
+
+export function getSameSpeciesEggCandyCounts(baseValue: integer): integer {
+  return starterCandyCosts[baseValue - 1].egg;
```

Comment:
```suggestion
export function getSameSpeciesEggCandyCounts(starterCost: integer): integer {
  return starterCandyCosts[starterCost - 1].egg;
```

Reply:
Adding: no more `integer` please:

```suggestion
export function getSameSpeciesEggCandyCounts(starterCost: number): number {
  return starterCandyCosts[starterCost - 1].egg;
```